src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Runtime.cs

```diff
@@ -7,18 +7,26 @@
 using System.Runtime.CompilerServices;
 using System.Threading.Tasks;
 
+using Console = System.Diagnostics.Debug;
+
 namespace System.Runtime.InteropServices.JavaScript
 {
     public static class Runtime
     {
-        private static readonly Dictionary<int, JSObject> _boundObjects = new Dictionary<int, JSObject>();
-        private static readonly Dictionary<object, JSObject> _rawToJS = new Dictionary<object, JSObject>();
-
-        // / <summary>
-        // / Execute the provided string in the JavaScript context
-        // / </summary>
-        // / <returns>The js.</returns>
-        // / <param name="str">String.</param>
+
+        private static readonly Dictionary<int, WeakReference> _boundObjects = new Dictionary<int, WeakReference>();
+        private static readonly Dictionary<object, JSObject?> _rawToJS = new Dictionary<object, JSObject?>();
```

Comment:
Please keep `Dictionary<object, JSObject?>` version no need to add nullable type